src/Conversion/ONNXToKrnl/NN/Pooling.cpp

```diff
@@ -132,17 +133,15 @@ void postProcessPoolingWindow<ONNXAveragePoolOp>(
   } else {
     denominator = poolDimValues[0];
     for (unsigned int i = 1; i < poolDimValues.size(); ++i)
-      denominator =
-          rewriter.create<arith::MulIOp>(loc, denominator, poolDimValues[i]);
+      denominator = create.math.mul(denominator, poolDimValues[i]);
     denominator = rewriter.create<arith::IndexCastOp>(
```

Comment:
use math cast here also. A grep will find them all, so I will stop comment on this